bower.json

```diff
@@ -12,6 +12,6 @@
     "docs"
   ],
   "dependencies": {
-    "d3": "3.4"
+    "d3": "3.5"
```

Comment:
I'd prefer this include the point version as well `3.5.5`.  Otherwise we might end up with breaking changes down the line.